packages/vscode-extension/src/webview/components/Preview.css

```diff
@@ -48,7 +48,6 @@
 
 .phone-screen {
   position: absolute;
-  width: calc(100% - 14px);
```

Comment:
AFAIK, this was needed for touch area margins. Does it break something when frame isn't visible?

Reply:
@jakub-gonet Right it's needed will be restored